src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -9703,6 +9659,66 @@ static bool isCandidateUnique(ref MethodSymbol? method, MethodSymbol candidate)
                 method = null;
                 return false;
             }
+
+            MethodSymbol? selectMethodForSignature(BoundMethodGroup node)
+            {
+                MethodSymbol? method = null;
+                if (node.ResultKind == LookupResultKind.Viable)
+                {
+                    foreach (var m in node.Methods)
```

Comment:
>foreach (var m in node.Methods) [](http://example.com/codeflow?start=20&length=31)

Do we have a test where there is a member  method and it is inaccessible?

Reply:
Yes, `MethodGroup_Inaccessible`. I'll also add `MethodGroup_ScopeByScope_InaccessibleInstance`